api/src/modules/h3-data/dto/h3-map-response.dto.ts

```diff
@@ -33,9 +33,24 @@ export class H3MapResponse {
           type: 'number',
         },
       },
+      indicatorDataYear: {
+        type: 'number',
+      },
+      harvestDataYear: {
+        type: 'number',
+      },
+      producerDataYear: {
+        type: 'number',
+      },
     },
   })
-  metadata: { unit: string; quantiles: number[] };
+  metadata: {
+    unit: string;
+    quantiles: number[];
+    indicatorDataYear?: number;
+    harvestDataYear?: number;
+    producerDataYear?: number;
```

Comment:
Even tho producer h3 data is required to do magic when it comes to risk-maps, it is just used internally to do said magic (and for some type of indicators) So I think we can safely remove it from the response

Also I think this data is included so the user can know with which years we sent a response, in case the we have no data for the requested year. So maybe avoid adding the value in case we have data for matching years? Or not worht the effort?

Reply:
Sure, I think also we can ask Elena and David about how they prefer to inform user about the the years situation, for us it is a minor detail of what to put in response